static/js/map.js

```diff
@@ -0,0 +1,251 @@
+// const modisURL = "modis.csv";
+const modisURL = "https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_modis";
```

Comment:
Make a note to remove these hardcoded urls (https://australia-fire-api-dashboard.herokuapp.com/api/v1.0) when we move the map to the flask app.

Fine for now. No need to do anything with this comment at the moment.

Reply:
@philipstubbs13 thanks for catching the "clear output" thing on the Jupyter notebook. those are a pain!

I'm now at a point where the map is functional when running on the Flask server. To your point above: is the best way to add the endpoing URLs for the map.js file to have them use relative locations for the endpoints? I'm not quite sure how to approach that.

Reply:
we can fix the endpoint urls tonight. I would just go ahead and merge what you have as long as it's working.